docker-entrypoint-prod.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+#echo "Running migrations (Database & static files to S3 bucket) ..."
+echo "Bucket: ${AWS_S3_BUCKET}"
+echo "Bucket User: ${AWS_S3_USER}"
+echo "Bucket ID: ${AWS_S3_KEYID}"...
+
+
+function send_message {
+    curl -s \
+      --form-string "token=avu1jex2ixynqx3waz92hwrjqdezrt" \
```

Comment:
Brian just explained to me that we only plan to use these tokens in the short run until we figure out how to load secret env vars into our Docker containers with some of the Heroku constraints. carry on...

Reply:
We are not using tokens, they work fine in Heroku and the migration process is responding well. The only challenge is that the migration process can take very long, and Heroku does have a 60 seconds boot timeout limit. If the service isn't live within 60 seconds, then it will take the app down.

Reply:
should or can we make the timeout longer?

Reply:
@mateoclarke we can increase the timeout but I don't think that's the best method to address this issue. The timeout is for booting up an already built container, so running migrations during that boot process is less than ideal. I addressed this issue earlier when we started getting timeouts when populating data from our yaml files in the entrypoint scripts.

The real solution should be to figure out how to use environment variables or other secure methods to store AWS keys etc. within the docker build process. That way we aren't trying to run it every time we boot up the container and we can keep our restart on failure times as short as possible.